readme.md

```diff
@@ -99,6 +99,10 @@ install target/release/choose <DESTDIR>
 
 Just make sure DESTDIR is in your path.
 
+### Packaging status
```

Comment:
This title should read "Installing From Package Managers" and there should be another sub title between line 86 and 88 that reads "Installing From Source".